deploy/Chart/templates/ucp/rbac.yaml

```diff
@@ -7,51 +7,32 @@ metadata:
     app.kubernetes.io/part-of: radius
 rules:
 - apiGroups:
-  - ""
+    - ""
   resources:
-  - configmaps
-  - secrets
-  - services
+    - secrets
   verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
+    - create
+    - delete
+    - get
+    - update
+
 - apiGroups:
-  - apps
+    - ucp.dev
   resources:
-  - deployments
-  - statefulsets
+    - resources
   verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
+    - create
+    - delete
+    - get
+    - list
+    - update
+
 - apiGroups:
-  - ucp.dev
+    - ucp.dev
   resources:
-  - resources
-  - queuemessages
+    - queuemessages
   verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
-- apiGroups:
-  - api.ucp.dev
-  resources:
-  - '*'
-  verbs:
-  - '*'
+    - create
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
```

Comment:
Removing this causes this error (an error that we have all seen before but didn't know why):

```
Radius release not found. Proceeding with installation.
*** Installing Radius to Kubernetes ***
Installing Radius version 8d1442d-dirty to namespace: radius-system...
*** Create workspace, group and environment for test ***
Creating workspace...
Set "kind-kind" as current workspace
Error: An unknown error was returned while testing Radius API status:
Status Code: 503
Response Headers:
  Audit-Id: 863b6ad6-02a3-4ba6-9c51-61f03dc66cf2
  Cache-Control: no-cache, private
  Content-Length: 20
  Content-Type: text/plain; charset=utf-8
  Date: Sun, 24 Nov 2024 16:36:20 GMT
  X-Content-Type-Options: nosniff
  X-Kubernetes-Pf-Flowschema-Uid: db65b004-bc64-4949-a3b2-2a0[80](https://github.com/radius-project/radius/actions/runs/11997783030/job/33443799631?pr=8080#step:18:81)1c4da00
  X-Kubernetes-Pf-Prioritylevel-Uid: fe22ed3a-e9f4-4f37-9901-d2347e099c15
Response Body:
service unavailable
```

Reply:
@ytimocin did you get a chance to look into the logs for error?

Reply:
Unfortunately, I missed that window. It is a resource permission issue but not exactly sure which one. I will keep an eye out for this error.